updates/3/updates.sql

```diff
@@ -70,11 +70,73 @@ INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x5273063725a43
+UPDATE assets SET coingecko="blockchain-certified-data-token" WHERE identifier="_ceth_0xAcfa209Fb73bF3Dd5bBfb1101B9Bc999C49062a5";/*Update coingecko*/
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xAcfa209Fb73bF3Dd5bBfb1101B9Bc999C49062a5", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xAcfa209Fb73bF3Dd5bBfb1101B9Bc999C49062a5", "C", "Blockchain Certified Data Token", "BCDT", 1513735057, NULL, "blockchain-certified-data-token", "BCDT", "0xAcfa209Fb73bF3Dd5bBfb1101B9Bc999C49062a5");
+UPDATE assets SET coingecko=NULL WHERE identifier="_ceth_0xAef38fBFBF932D1AeF3B808Bc8fBd8Cd8E1f8BC5";/*Update coingecko*/
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xAef38fBFBF932D1AeF3B808Bc8fBd8Cd8E1f8BC5", 8, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xAef38fBFBF932D1AeF3B808Bc8fBd8Cd8E1f8BC5", "C", "Creditbit", "CRB", 1446595200, NULL, NULL, "CRB", "0xAef38fBFBF932D1AeF3B808Bc8fBd8Cd8E1f8BC5");
+UPDATE assets SET coingecko=NULL WHERE identifier="_ceth_0x662aBcAd0b7f345AB7FfB1b1fbb9Df7894f18e66";/*Update coingecko*/
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x662aBcAd0b7f345AB7FfB1b1fbb9Df7894f18e66", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x662aBcAd0b7f345AB7FfB1b1fbb9Df7894f18e66", "C", "CarTaxi Token", "CTX", 1506211200, NULL, NULL, "CTX", "0x662aBcAd0b7f345AB7FfB1b1fbb9Df7894f18e66");
+UPDATE assets SET coingecko=NULL WHERE identifier="_ceth_0x497bAEF294c11a5f0f5Bea3f2AdB3073DB448B56";/*Update coingecko*/
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x497bAEF294c11a5f0f5Bea3f2AdB3073DB448B56", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x497bAEF294c11a5f0f5Bea3f2AdB3073DB448B56", "C", "DEX", "DEX", 1540376115, NULL, NULL, "DEX", "0x497bAEF294c11a5f0f5Bea3f2AdB3073DB448B56");
+UPDATE assets SET coingecko=NULL WHERE identifier="_ceth_0x4DF47B4969B2911C966506E3592c41389493953b";/*Update coingecko*/
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x4DF47B4969B2911C966506E3592c41389493953b", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x4DF47B4969B2911C966506E3592c41389493953b", "C", "FundRequest", "FND", 1540376115, NULL, NULL, "FND", "0x4DF47B4969B2911C966506E3592c41389493953b");
+UPDATE assets SET coingecko="smart-mfg", name="Smart MFG" WHERE identifier="_ceth_0x6710c63432A2De02954fc0f851db07146a6c0312";/*Update coingecko*/
```

Comment:
According to the contract the name is: "SyncFab Smart Manufacturing Blockchain". Why are you changing it here?
Also you will need to update the cryptocompare entry:

```
sqlite> SELECT * from assets where identifier ="_ceth_0x6710c63432A2De02954fc0f851db07146a6c0312";
_ceth_0x6710c63432A2De02954fc0f851db07146a6c0312|C|SyncFab|MFG|1508284800||syncfab||0x6710c63432A2De02954fc0f851db07146a6c0312
sqlite> 
```